src/hotspot/cpu/ppc/continuationFreezeThaw_ppc.inline.hpp

```diff
@@ -264,15 +264,15 @@ frame FreezeBase::new_heap_frame(frame& f, frame& caller) {
 
   intptr_t *sp, *fp;
   if (FKind::interpreted) {
-    int locals = f.interpreter_frame_method()->max_locals();
+    intptr_t offset = *f.addr_at(ijava_idx(locals));
```

Comment:
I'd prefer a more specific name. Maybe `locals_offset` or `local0_offset`?

Reply:
Changed the name to `locals_offset` (in all platforms).